.github/workflows/deploy-prod-static.yml

```diff
@@ -40,13 +52,26 @@ jobs:
           cp -r packages/typescriptlang-org/public site
         env:
           YARN_CHECKSUM_BEHAVIOR: ignore
+      
+      - name: Setup Pages
+          uses: actions/configure-pages@983d7736d9b0ae728b81ab479565c72886d7745b # v5
+          if: github.event_name != 'pull_request'
 
-      - name: Deploy
-        id: builddeploy
-        uses: Azure/static-web-apps-deploy@v1
```

Comment:
Leave this deployment in; we want the existing site to keep working in the meantime.

Reply:
And I would put this above the new gh-pages stuff in order, so the existing side deploys before gh-pages in case that goes wrong.